### PR TITLE
Added init to WinkEggTray

### PR DESCRIPTION
**Pull request by w1ll1am23**
Had to add in the init method for WinkEggTray for updates to function.

### CODE REVIEW

File: pywink/__init__.py [hunk 1]
```diff
@@ -161,6 +161,9 @@ class WinkEggTray(WinkDevice):
 }
 
 """
+    def __init__(self, device_state_as_json, objectprefix="eggtrays"):
+        super(WinkEggTray, self).__init__(device_state_as_json,
+                                            objectprefix=objectprefix)
```
Comment (bradsk88) [inline pywink/__init__.py]: super minor, but `objectprefix` should line up with the start of `device_state_as_json`

**Comment by bradsk88** [comment]
Don't worry about travis check

**Comment by bradsk88** [comment]
Thanks.  Sorry I wiped that out.

**Comment by bradsk88** [comment]
Should probably update the version and changelog as well.

**Comment by w1ll1am23** [comment]
Version to 3.0.1?

**Comment by bradsk88** [comment]
@w1ll1am23 0.3.1

**Comment by bradsk88** [comment]
I won't be able to push this up to pypi for a couple of hours (3)

**Comment by bradsk88** [comment]
It's now on pypi

**Pull request merged**